### PR TITLE
Fix Dapr update info

### PR DESCRIPTION
**Pull request by marcduiker**
Specify 1.11.0 where needed and fix link.

### CODE REVIEW

File: daprblog/content/posts/2023/v1.11-release.md [hunk 1]
```diff
@@ -492,16 +492,16 @@ For RC releases like this, download the latest and greatest release from [here](
 Once you have installed the CLI, run:
```
Comment (paulyuk) [inline daprblog/content/posts/2023/v1.11-release.md:492]: Suggest we include a step to get the CLI.  We could point off to doc for this.

File: daprblog/content/posts/2023/v1.11-release.md
Comment (paulyuk) [inline daprblog/content/posts/2023/v1.11-release.md:1]: Line 490 (not part of PR yet) suggest we strike "for RC releases like this".

**Review by paulyuk — CHANGES_REQUESTED** [review]
Please see comments

**Review by yaron2 — APPROVED** [review]

**Review by artursouza — APPROVED** [review]

**Comment by hhunter-ms** [comment]
@marcduiker for the blog repo, PRs only build successfully if created from dapr/blog, not from a fork of dapr/blog... try creating a branch on dapr/blog (not marcduiker/dapr-blog) and opening a new PR

**Pull request closed without merging**